src/cfn/template.yaml

```diff
@@ -18,12 +19,25 @@ Globals:
       AllowOrigin: "'*'"
 Parameters:
   CollectionId:
+    Description: AWS Resources are named based on the value of this parameter. You must customise this if you are launching more than one instance of the stack within the same account.
     Type: String
     Default: RekogDemo
     AllowedPattern : ^[a-zA-Z0-9_]*$
+  IncludeCloudFrontDistribution:
```

Comment:
I think 'Create' may be a better verb than 'Include' here. To me it'd make the parameter purpose and acceptable values more obvious. What do you think?

Reply:
Ok done